Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Prefabs/HandMenu_Slider_Assets/Scripts/UpdateTrackScale.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.UI;
+
+
+public class UpdateTrackScale : MonoBehaviour
```

Comment:
Is this intended to be an example script, or something that consumers will use?

If it's more of the latter, we'd probably want to expose roundedValue/trackScaleNode publicly as well so that these things are changeable at runtime by code.

Reply:
The script is more just to make sure the active part of the track is updated based on the current level of the slider. I wouldn't say it should be customizable.

Reply:
Gotcha.

I think that in this case we should have the fields commented (i.e

```csharp
// A value in the range [0, 100] indicating the slider position
... roundedValue...
```

Reply:
Thanks! I just updated the script and removed the roundedValue altogether. It now references eventData.NewValue, which is more straightforward.